src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -166,6 +166,33 @@ message Capacity {
     uint32 desired = 3;
 }
 
+/// Job disruption budget, associated (optionally) with a job.
+message JobDisruptionBudget {
+
+    /// The minimum required percentage of tasks in a healthy state. Tasks will not be terminated by the eviction service
+    //  if this limit would be violated.
+    message AvailabilityPercentageLimit {
+        double limit = 1;
+    }
+
+    /// The minimum required amount of tasks in a healthy state. Tasks will not be terminated by the eviction service
+    //  if this limit would be violated.
+    message AvailableTasksLimit {
+        uint32 limit = 1;
+    }
+
+    /// Maximum number of times a task can be evicted.
+    message EvictionLimit {
+        uint32 limit = 1;
+    }
+
+    oneof Policy {
```

Comment:
we may need a way to compose these policies. Is it worth future-proofing it a bit and adding a `ComposableJobDisruptionBudget`?

Reply:
This is only initial proposal, that is not referenced yet by anything. I would like to work more on the job disruption budget concept in the next quarter.